### ISSUE TITLE
Could not load xlwings32-dev.dll error When trying to use UDF

### ISSUE THREAD

**Issue by Adel-Moustafa** [xlwings/xlwings#2098]
#### OS (e.g. Windows 10 or macOS Sierra)
Windows 10
#### Versions of xlwings, Excel and Python (e.g. 0.11.8, Office 365, Python 3.7)
- xlwings : ` 0.27.15`
- Excel: `2013`
- Python: ` 3.9.13` 
#### Describe your issue (incl. Traceback!)
Hi, I have installed Anaconda3-2022. and I tried to run python code from excel using VBA just like this [example](https://docs.xlwings.org/en/latest/vba.html) and it works just fine. but the problem arises when I try to use UDF, I have followed the instruction in the documentation under **[User Defined Functions (UDFs)](https://docs.xlwings.org/en/latest/udfs.html)** it gives me the following error:    

Run-time error '1':
Could not load xlwings32-dev.dll from either of the following folders:
C:\ProgramData\Anaconda3
C:\ProgramData

**note:** I am not the administrator of my machine because it belongs to the company that I work in.

thanks.

**Comment by fzumstein** [comment]
It looks like you have installed the xlwings addin directly from the source code repo (dev version), instead of using the `xlwings addin install` command.

**Comment by Adel-Moustafa** [comment]
yes this exactly what I did, then I copied the file `xlwings.xlam` to the add-ins folder in excel.
If I install xlwings from anaconda prompt using `xlwings addin install` would that fix the problem ?

**Comment by fzumstein** [comment]
> If I install xlwings from anaconda prompt using xlwings addin install would that fix the problem ?
yes

See: https://docs.xlwings.org/en/stable/installation.html#add-in

**Comment by Adel-Moustafa** [comment]
Thank you very much for your support, it did solve the problem.
